submission/submission-security/src/main/kotlin/ebi/ac/uk/security/service/CaptchaVerifier.kt

```diff
@@ -0,0 +1,26 @@
+package ebi.ac.uk.security.service
+
+import ebi.ac.uk.security.integration.SecurityProperties
+import ebi.ac.uk.security.integration.exception.InvalidCaptchaException
+import org.springframework.web.client.RestTemplate
+import org.springframework.web.client.postForObject
+import java.net.URI
+
+internal val VERIFY_URL = URI.create("https://www.google.com/recaptcha/api/siteverify")
+
+internal class CaptchaVerifier(
+    private val restTemplate: RestTemplate,
+    private val properties: SecurityProperties
+) {
+    fun verifyCaptcha(userKey: String?) {
+        if (userKey.isNullOrBlank()) throw InvalidCaptchaException()
```

Comment:
We should use `require` here to be consistent with other validations in the system